.github/workflows/test_node.yml

```diff
@@ -19,10 +24,13 @@ jobs:
         with:
           node-version-file: package.json
 
-      # We need to skip the fallback download because downloading will fail on release branches because the new version isn't available yet.
-      # We have to use npm here because yarn fails on the non-existing existing optionalDependency version:
-      # https://github.com/yarnpkg/berry/issues/2425#issuecomment-1627807326
-      - run: SENTRYCLI_SKIP_DOWNLOAD=1 npm ci
+      - name: Install dependencies via npm ci
+        if: ${{ !inputs.triggered-by-release }}
+        run: npm ci
```

Comment:
didn't we previously just do `SENTRYCLI_SKIP_DOWNLOAD=1 npm install`? Seemed a bit simpler than the check and the conditional steps 😅 

No need to change if this works, was just curious.

Reply:
l: Also, do we still need the `SENTRYCLI_SKIP_DOWNLOAD` env variable check at all anymore?

Reply:
> didn't we previously just do SENTRYCLI_SKIP_DOWNLOAD=1 npm install? Seemed a bit simpler than the check and the conditional steps 😅

Yeah, I think that is why it broke now, but at the same time, I want the locking to avoid situations like we had a few days ago, with incompatible releases breaking our CI. Better to run CI against locked dependencies to make it reproducible, imo

Reply:
We probably could remove the check, only problem is we advertise it as public API in the `README.md`